lib/at_host/at_host.c

```diff
@@ -39,14 +39,28 @@ static const struct device *const uart_dev = DEVICE_DT_GET(DT_CHOSEN(ncs_at_host
 #else
 static const struct device *const uart_dev = DEVICE_DT_GET(DT_NODELABEL(uart0));
 #endif
+
+#if DT_HAS_CHOSEN(zephyr_log_uart)
+static const bool host_uart_is_log_uart = \
+	(uart_dev == DEVICE_DT_GET(DT_CHOSEN(zephyr_log_uart)));
```

Comment:
This line is within 100 chars, no need to split it.
```suggestion
static const bool host_uart_is_log_uart = (uart_dev == DEVICE_DT_GET(DT_CHOSEN(zephyr_log_uart)));
```